samples/crypto/ecdsa/README.rst

```diff
@@ -7,23 +7,27 @@ Crypto: ECDSA
    :local:
    :depth: 2
 
+The ECDSA sample shows how to sign and verify messages using SHA-256 as the hashing algorithm and the secp256r1 curve.
+
 Overview
 ********
 
-The ECDSA sample shows how to sign/verify messages using SHA256 as the hashing algorithm and the secp256r1 curve.
+The sample follows these steps:
+
+1. First, the sample performs initialization:
 
+   a. The Platform Security Architecture (PSA) API is initialized.
+   #. A random ECDSA key pair is generated and imported into the PSA crypto keystore.
+   #. The public key of the ECDSA key pair is imported into the PSA crypto keystore.
 
-First, the sample performs initalization:
-   #. The Platform Security Architecture (PSA) API is initialized.
-   #. A random ECDSA keypair is generated an imported in the PSA crypto keystore
-   #. The public key of the ECDSA keypair is imported in the PSA crypto keystore
+#. Then, ECDSA signing and verification is performed:
 
-Then, ECDSA signing/verification is performed:
-   #. Signing is performed using the ECDSA keypair.
-   #. Verification of the signagure is performed using the exported public key.
+   a. Signing is performed using the ECDSA key pair.
```

Comment:
Signing is performed with the ECC private key

Reply:
That is correct of course, but I don't create a private key in my code. I create a key pair (which obviously include the private key), so I wanted to be more consistent with the code. We can change that to:
 
  a. Signing is performed using the private key in the ECDSA key pair.

Reply:
Changed to suggestion above